src/main/java/com/kryptnostic/rhizome/mappers/keys/ByteArrayKeyMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.kryptnostic.rhizome.mappers.keys;
+
+import com.kryptnostic.rhizome.mappers.SelfRegisteringKeyMapper;
+
+public class ByteArrayKeyMapper implements SelfRegisteringKeyMapper<byte[]> {
+
+    @Override
+    public String fromKey( byte[] key ) {
+        return key == null ? null : new String( key );
```

Comment:
Sounds good to me (UTF_8 unless @makosblade or @geekbeast have a good reason for preferring another)